core/deployment/src/main/java/io/quarkus/runner/RuntimeClassLoader.java

```diff
@@ -522,14 +526,22 @@ private ProtectionDomain createDefaultProtectionDomain(Path applicationClasspath
         URL url = null;
         if (applicationClasspath != null) {
             try {
-                URI uri = new URI("file", null, applicationClasspath.toString(), null);
+                // String path = applicationClasspath.toString();
+                //if (File.separatorChar != '/') {
+                //    // Note that windows separator is always quoted in the URI constructor
+                //    path = path.replace(File.separatorChar, '/');
+                //}
+                //URI uri = new URI("file", null, path, null);
+                URI uri = applicationClasspath.toUri();
```

Comment:
Is it normal that we have all this commented code here (and one line below)?

Reply:
No. It's a leftover from a fix I did and wasn't quite sure if it's correct. I'll remove the commented code.